<launch>

<!-- Parameters -->
 <include file="$(find door_handle_detector)/launch/door_handle_detector_params.xml"/>

<!-- Handle detector Camera 
 <node pkg="door_handle_detector" name="handle_detector_vision" type="handle_detector_vision" respawn="true" output="screen">
     <param name="display" value="False" />
 </node>
-->

<!-- Handle detector Laser -->
  <node pkg="door_handle_detector" type="handle_detector_laser" name="handle_detector_laser" respawn="true" output="screen">
   <remap from="/full_cloud" to="/full_cloud_cropped"/>
  </node>

<!-- Door detector -->
  <node pkg="door_handle_detector" type="doors_detector_laser" name="doors_detector_laser" respawn="true" output="screen"/>

</launch>

